articles/quickstart/native/ios-swift/03-user-sessions.md

```diff
@@ -187,10 +197,13 @@ Auth0
 You can add custom user information in the user metadata section by performing a `patch`:
 
 ```swift
+// ProfileViewController.swift
+
 let accessToken = ... // You will need the accessToken from your credentials instance 'credentials.accessToken'
 let profile = ... // the Profile instance you obtained accessing the `/userinfo` endpoint.
+
 Auth0
-    .users(token: "user-scoped access token")
+    .users(token: accessToken)
```

Comment:
The one problem with changing this is that the "user-scoped" message is lost (though this is much more clear overall). Maybe just a code comment that mentions the access token needs to be issued on behalf of the user.

Reply:
One more below

Reply:
I've changed the constant definition to `let accessToken = ... // the user-scoped accessToken from your credentials instance 'credentials.accessToken'`.